docs/recipes/Module-Composition.md

```diff
@@ -6,6 +6,8 @@
 
 A key part of working with One App is module composition. Modules can be rendered inside one another by either using `ModuleRoute` or `RenderModule`.
 
+> Modules that you plan on rendering need to be added to your [module map](../api/server/Module-Map-Schema.md) before they can be rendered.
```

Comment:
What about mentioning in development you need to serve the modules (one app runner array or `serve-module`) and in production your has to be deployed to your module map?